app/presenters/region_presenter.rb

```diff
@@ -158,15 +149,25 @@ def top_gl_coverage_countries
     }
   end
 
+  def get_gl_data_for_countries(countries)
+    countries.map do |country|
+      total_area = country.country_statistic.total_area
+      total_gl_area = country.total_gl_coverage
+      percentage_of_total_area = ((total_gl_area / total_area).to_f * 100).round(1)
+      
+      { country: country, total_area: total_gl_area, percentage: percentage_of_total_area }
+    end.sort! do |a, b|
+      # If rounded %s happen to be the same, then sort by area
+      b[:percentage] == a[:percentage] ? b[:total_area] <=> a[:total_area] : b[:percentage] <=> a[:percentage]
+    end.take(10)
```

Comment:
I think `take` doesn't take into account the order of the elements in the array. I would use `first(10)` to be sure.
The overall method could perhaps benefit of a SQL version refactoring but that's ok for now

Reply:
Did a test: 
```
irb(main):001:0> array = [1, 2, 3, 4, 5]
=> [1, 2, 3, 4, 5]
irb(main):002:0> array.take(2)
=> [1, 2]
irb(main):003:0> array.first(2)
=> [1, 2]
```
Looks to be the same to me?

Reply:
Yeah so it should be fine with arrays; order is not preserved when using take with AR models apparently. So that's ok :)